07_If_elif_else.py

```diff
@@ -0,0 +1,34 @@
+"""
+Hometask 07 if_elif_else.
+
+# given values w,x,y,z
+# write if-elif-else statement that will search
+minimum value and print smth aka "'y' is minimum value'
+# where 'y' is variable name (not value)
+# advice use python debugger and different values to check your algorithm
+# optionally you can check your algorithm somehow with assert statements
+
+"""
+
+w, x, y, z = 100, 200, 40, 300
+
+if w > x:
+    wx = x
+else:
+    wx = w
+if y > z:
+    yz = z
+else:
+    yz = y
+if (wx == yz) or (w == x) or (y == z):
```

Comment:
Try to improve your algorithm and output in case of equal values first found variable name.
For example if we have  w, x, y, z = 0, -1, 0, -1  our target result will be 'x'